src/summary/history/allergy-card-level-three.css

```diff
@@ -0,0 +1,61 @@
+.allergyHeader {
+  margin: 0rem 0rem 0.25rem 1rem;
+  color: var(--omrs-color-ink-medium-contrast);
+}
+
+.container {
+  justify-content: center;
+  display: flex;
+  background: var(--omrs-color-ink-white);
+  padding-left: 1rem;
+  flex-direction: column;
+  border: 0.0625rem solid var(--omrs-color-ink-low-contrast);
+  margin: 0rem 1rem 0.5rem 1rem;
+  padding: 1rem;
+  max-height: 250px;
+  flex-wrap: wrap;
+}
+
+.container div {
+  margin-bottom: 1rem;
+}
+
+.container div:last-child {
+  margin-bottom: 0rem;
+}
+
+.allergyCommentContainer {
+  display: flex;
+  background: "var(--omrs-color-bg-medium-contrast)";
+  margin: 0rem 1rem 4.5rem 1rem;
+}
+
+.allergyCommentTextArea {
+  outline: none;
+  width: 100%;
+  padding: 0.5rem;
+}
+
+.allergyCommentTextArea:valid {
+  color: var(--omrs-color-interaction);
+}
+
+.buttonStyles {
+  display: flex;
+  background: white;
+  bottom: 0;
+  justify-content: center;
+  position: absolute;
```

Comment:
Again, why use absolute?

Reply:
![Screenshot from 2019-12-12 17-13-12](https://user-images.githubusercontent.com/28008754/70719279-c0341400-1d02-11ea-9ba9-85700d7de88a.png)
When `position: absolute` is used to make the element stick to the bottom of the screen.

![Screenshot from 2019-12-12 17-13-25](https://user-images.githubusercontent.com/28008754/70719369-eeb1ef00-1d02-11ea-9ab9-bcb875894f81.png)
when the `position: absolute` is not used the behavior is seen above

Reply:
that makes sense.